docs/processes/coding standard/Wordpress/WPCS.md

```diff
@@ -0,0 +1,178 @@
+
+        -------------------------------------------------------
+
+        FOUND 35 ERRORS AND 157 WARNINGS AFFECTING 1997 LINES
+
+        -------------------------------------------------------
+        4 | WARNING | Found precision alignment of 1 spaces.
+
+        9 | ERROR   | Missing file doc comment
+
+        9 | ERROR   | All output should be run through an escaping function
+
+        ``` 
+
+  7. To check all phpcs flags use this Command: `phpcs –help`
+
+ ## Dynamically Run Phpcs and Phpcbf command on saving the php file
+
+  1. Install [Run on Save](https://marketplace.visualstudio.com/items?itemName=emeraldwalk.RunOnSave) extension by ***emeraldwalk*** on Vscode
+  2. As Vscode setting is displayed in UI format we have to change it to JSON format
+    - Go To -` Vscode -> setting -> Workbench - setting Editor -> And change the Editor option to JSON`
+
+      ```json
+        "emeraldwalk.runonsave": {
+            "commands": [
+                {
+                    "match": "\\.php$",
+                    "cmd": "phpcbf --standard='WordPress' ${file}",
+                    "cmd": "phpcs --standard='WordPress' ${file}",
+                },
+                {
+                  "match": "\\.php$",
+                  "cmd": "rm ${workspaceFolder}/errors/${fileBasenameNoExt}",
+                  "cmd": "phpcs --standard='WordPress' -a ${file} > ${workspaceFolder}/errors/${fileBasenameNoExt}.txt"
+                }
+            ]
+          }
+      ```
+
+      :::tip How to check PHPCS error
+      *After adding this restart vscode now on saving the .php file it will run added phpcs command. Now it will start throwing error related to wordpress.*
+      *For checking error in vscode go to `Terminal -> output -> on right hand side there will be a dropdown open it and select Run on Save` You will get the error here.*
+      :::
+## Validating WP Functions
+
+  1. Add [PHP Intelephense](https://marketplace.visualstudio.com/items?itemName=bmewburn.vscode-intelephense-client) Vscode extension by *Ben Mewburn*
+  2. For enabling this extension for Wordpress go to <mark>Vscode -> setting -> Extension -> Intelephense -> Intelephense Stubs</mark> and click on add item and add ***Wordpress***.
+  3. As this function are not wordpress specific. It will give error for acf methods like get_field, have_rows.
+  4. For removing the above error clone this [acf-pro-stubs](https://github.com/php-stubs/acf-pro-stubs) in your document directory
+  5. Go to Vscode setting in JSON format add the acf-pro-stubs directory for PHP Intelephense
+
+       ```array
+        "intelephense.stubs": [ 
+          “WordPress”,
+          ”/var/www/html/stubs/acf-pro-stubs”,
+        ]
+        ```
+  - Now the ACF methods error will be gone.
+
+## Important Extension for Development
+
+  - Using [acf-snippets](https://marketplace.visualstudio.com/items?itemName=anthonydiametrix.ACF-Snippet) vscode extension by *Anthony Hubble*
+    1. field:if / field:ifelse
```

Comment:
Please put more light on this statements what are they how to be used in the development